<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>DVKBuntu Acceuil</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/Logo.png</normaloff>:/images/Logo.png</iconset>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
background-image: rgba(255, 255, 255, 0);
}</string>
   </property>
   <widget class="QPushButton" name="OuvrirNavigateur">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>220</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Lancement &lt;/p&gt;&lt;p&gt;du navigateur &lt;/p&gt;&lt;p&gt;par défaut&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:rgb(0, 0, 0)
}</string>
    </property>
    <property name="text">
     <string>Naviguer 
Sur le net</string>
    </property>
    <property name="icon">
     <iconset theme="internet-web-browser">
      <normaloff>.</normaloff>.</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="OuvrirKmag">
    <property name="geometry">
     <rect>
      <x>259</x>
      <y>210</y>
      <width>220</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;body&gt;Ouverture &lt;/span&gt;&lt;/p&gt;
&lt;p&gt;de la loupe&lt;/span&gt;&lt;/p&gt;
&lt;p&gt;d'écran&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:rgb(0, 0, 0)
}</string>
    </property>
    <property name="text">
     <string>Ouvrir 
Kmag</string>
    </property>
    <property name="icon">
     <iconset theme="kmag">
      <normaloff>.</normaloff>.</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="ConfigORCA">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>210</y>
      <width>220</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Lancer la&lt;/p&gt;&lt;p&gt;Configuration&lt;/p&gt;&lt;p&gt;des paramètres &lt;/p&gt;&lt;p&gt;d'accessibilités&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:rgb(0, 0, 0)
}</string>
    </property>
    <property name="text">
     <string>Accès
Handicap</string>
    </property>
    <property name="icon">
     <iconset theme="orca">
      <normaloff>.</normaloff>.</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="OuvrirSiteWeb">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>220</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ouverture&lt;/p&gt;&lt;p&gt;du site web&lt;/p&gt;&lt;p&gt;DVKBuntu&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:rgb(0, 0, 0)
}</string>
    </property>
    <property name="text">
     <string>Ouvrir
Site Web</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/images/Logo.png</normaloff>:/images/Logo.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Discord">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>210</y>
      <width>220</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Accès au&lt;/p&gt;&lt;p&gt;Discord de&lt;/p&gt;&lt;p&gt;DVKBuntu&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:rgb(0, 0, 0)
}</string>
    </property>
    <property name="text">
     <string>Rejoindre
Discord </string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/images/Discord.svg</normaloff>:/images/Discord.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="NoComprendo">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>330</y>
      <width>220</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Lancement&lt;/p&gt;&lt;p&gt;Assistant Vocal&lt;/p&gt;&lt;p&gt;NoComprendo&lt;/p&gt;&lt;p&gt;Pour DVKBuntu&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:rgb(0, 0, 0)
}</string>
    </property>
    <property name="text">
     <string>Assistant
Vocal</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/images/NoComprendo.jpg</normaloff>:/images/NoComprendo.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Contacts">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>450</y>
      <width>220</width>
      <height>100</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:rgba(120, 120, 120, 120)
}</string>
    </property>
    <property name="text">
     <string>Contactez 
nous</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/images/Mail.svg</normaloff>:/images/Mail.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>450</y>
      <width>460</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background: rgba(0, 0, 0, 180);
}</string>
    </property>
    <property name="text">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; text-decoration: underline; color:#ffffff;&quot;&gt;Bienvenue à tous sur DVKBuntu, &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-style:italic; color:#ffffff;&quot;&gt;L'équipe HandyOpenSource (HandyOS).&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#ffffff;&quot;&gt;&lt;br /&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QSlider" name="ScaleFactor">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>521</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider{
background:grey;
}</string>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>30</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>18</number>
    </property>
    <property name="sliderPosition">
     <number>18</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBothSides</enum>
    </property>
   </widget>
   <widget class="QLabel" name="ScaleFactorValue">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>521</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background:rgb(0, 0, 0)
}</string>
    </property>
    <property name="text">
     <string>Changer facteur  d'échelle</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Power">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>30</y>
      <width>176</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="system-shutdown">
      <normaloff>.</normaloff>.</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>144</width>
      <height>144</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Reboot">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>190</y>
      <width>176</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="system-reboot">
      <normaloff>.</normaloff>.</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>144</width>
      <height>144</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Deconnection">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>360</y>
      <width>176</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="system-log-out">
      <normaloff>.</normaloff>.</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>144</width>
      <height>144</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Forum">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>220</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ouverture&lt;/p&gt;&lt;p&gt;du forum&lt;/p&gt;&lt;p&gt;DVKBuntu&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:rgb(0, 0, 0)
}</string>
    </property>
    <property name="text">
     <string>Ouvrir
le Forum</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/images/Forum.png</normaloff>:/images/Forum.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="VocalFly">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>330</y>
      <width>220</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Lancement&lt;/p&gt;&lt;p&gt;interface de&lt;/p&gt;&lt;p&gt;gestion de VocalFly&lt;/p&gt;&lt;p&gt;Pour DVKBuntu&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:rgb(0, 0, 0)
}</string>
    </property>
    <property name="text">
     <string>VocalFly</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/images/vocalflyicon.png</normaloff>:/images/vocalflyicon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="GitHub">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>330</y>
      <width>220</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Voir le code&lt;/p&gt;&lt;p&gt;source de&lt;/p&gt;&lt;p&gt;DVKBuntu&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:rgb(0, 0, 0)
}</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhLatinOnly</set>
    </property>
    <property name="text">
     <string>Voir le 
Code</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/images/GitHub.png</normaloff>:/images/GitHub.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="Image">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>1241</width>
      <height>621</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/DVKBuntu-etoiles-filantes.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>Image</zorder>
   <zorder>OuvrirNavigateur</zorder>
   <zorder>OuvrirKmag</zorder>
   <zorder>ConfigORCA</zorder>
   <zorder>OuvrirSiteWeb</zorder>
   <zorder>Discord</zorder>
   <zorder>NoComprendo</zorder>
   <zorder>Contacts</zorder>
   <zorder>label</zorder>
   <zorder>ScaleFactor</zorder>
   <zorder>ScaleFactorValue</zorder>
   <zorder>Power</zorder>
   <zorder>Reboot</zorder>
   <zorder>Deconnection</zorder>
   <zorder>Forum</zorder>
   <zorder>VocalFly</zorder>
   <zorder>GitHub</zorder>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
